### PR TITLE
Add CSS icon

### PR DESCRIPTION
**Pull request by LitoMore**
**Popularity metric:** Popular enough no doubt

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

I'm adding this as a new icon. The `CSS3` in our collection can be removed from another PR.

Related:

- https://github.com/CSS-Next/css-next/issues/105
- https://github.com/CSS-Next/logo.css/issues/23

### CODE REVIEW

**Comment by uncenter** [comment]
There were some minor but adjustments nonetheless made to the "CSS" lettering yesterday after you made this: https://github.com/CSS-Next/logo.css/commit/9e25cf1ea59f60d0601253a71c23c36c7db4819e. Would you mind updating to the latest version of the source?

**Comment by LitoMore** [comment]
@uncenter Updated.

File: icons/css.svg
Comment (uncenter) [inline icons/css.svg:1]: Seeing differences in the lettering still.
CleanShot 2024-11-15 at 20 38 41

**Review by uncenter — CHANGES_REQUESTED** [review]

File: icons/css.svg
Comment (LitoMore) [reply icons/css.svg:1]: I'm using the `css.svg` one. Which one did you use?

Comment (uncenter) [reply icons/css.svg:1]: The one linked as source. Went to the link, clicked download, opened your version of the SVG, added in the downloaded SVG. Are you sure you have updated the source SVG file on your end?

File: _data/simple-icons.json [hunk 1]
```diff
@@ -3849,6 +3849,12 @@
             "hex": "FC5531",
             "source": "https://www.csdn.net/company/index.html"
         },
+        {
+            "title": "CSS",
+            "hex": "663399",
+            "source": "https://github.com/CSS-Next/logo.css/blob/4e5bae2d287e5523dc30cbdd8d9f4885b307adc7/css.svg",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:3855]:
```suggestion
            "source": "https://github.com/CSS-Next/logo.css/blob/bacc20878227204b283c68a6b935f8279e06b0cd/css.svg",
```
Source changed. Attaching the wire frame of the comparison as a screenshot.
image

**Comment by uncenter** [comment]
Might want to just let this one set for a minute until they stop making frequent updates to it 😅

**Comment by LitoMore** [comment]
We can update it anytime. Don't be blocked by them.

**Comment by uncenter** [comment]
> This is also annoying for me every time I update this PR from their change.

Right, so I'm saying maybe we can take a pause for a bit and just let any other changes accumulate instead of repeating this 4 more times. I don't want to merge this and have an update PR tomorrow.

**Review by PratyushJayachandran — APPROVED** [review]

**Pull request merged**